packages/apps/reputation-oracle/server/src/modules/abuse/abuse.service.ts

```diff
@@ -225,18 +241,30 @@ export class AbuseService {
           abuseEntity.chainId,
           abuseEntity.escrowAddress,
         );
+        if (!escrow) {
+          throw new Error(
+            `Escrow data is missing. Abuse id: ${abuseEntity.id}`,
+          );
```

Comment:
```suggestion
          throw new Error('Escrow data is missing');
```
and for `catch` of loop iterations:
```
this.logger.error('Error sending abuse', {
   error: error,
   abuseId: abuseEntity.id,
});
```